src/community/geomesa/pom.xml

```diff
@@ -0,0 +1,218 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!--
+Copyright (C) 2015 - Open Source Geospatial Foundation. All rights reserved.
+This code is licensed under the GPL 2.0 license, available at the root
+application directory.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.geoserver</groupId>
+    <artifactId>community</artifactId>
+    <version>2.8-SNAPSHOT</version>
+  </parent>
+
+  <groupId>org.geoserver.community</groupId>
+  <artifactId>gs-web-geomesa</artifactId>
+  <packaging>jar</packaging>
+  <version>2.8-SNAPSHOT</version>  
+  <name>GeoMesa Web Module</name>
+

```

Comment:
please add a description and developer contact info - as it can help you generate new biz leads and collaborators